tests/schemes/test_json.py

```diff
@@ -4,37 +4,26 @@
 from __future__ import unicode_literals
 
 import pytest
-import tornado
 
-from tchannel import TChannel
-from tchannel import response
-from tchannel import schemes
+from tchannel import TChannel, Response, schemes
+from tchannel.response import ResponseTransportHeaders
 
 
 @pytest.mark.gen_test
-@pytest.mark.call
+@pytest.mark.callz
```

Comment:
todo - rename all these mark `call` as `pytest.mark.public` so that we can quickly check the integration tests related to the public api